internal/core/metadata/rest_deviceprofile.go

```diff
@@ -457,12 +473,16 @@ func restGetYamlProfileById(w http.ResponseWriter, r *http.Request) {
 }
 
 // Notify the associated device services for changes in the device profile
-func notifyProfileAssociates(dp models.DeviceProfile, dl device.DeviceLoader, action string) error {
+func notifyProfileAssociates(
+	dp models.DeviceProfile,
+	dl device.DeviceLoader,
+	action string,
+	loggingClient logger.LoggingClient) error {
```

Comment:
Calling out once; may be other instances.  When folding function/method definitions, to assist readability please leave an empty line after the function's/method's definition and before the first line of the function's/method's implementation.